tools/yamcan/classes/Can.py

```diff
@@ -326,7 +400,7 @@ def calc_signal_params(self):
                     sig_range = nat_rep.range.max - nat_rep.range.min
                 elif nat_rep.signedness == Signedness.signed:
                     self.offset = 0
-                    sig_range = 2* max(nat_rep.range.max, nat_rep.range.min)
+                    sig_range = 2* max(nat_rep.range.max, -nat_rep.range.min)
```

Comment:
This is only true if min is negative, although i think you did catch a bug here. nice